### DISCUSSION TITLE
NexoPOS 4.5.0

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#292] [Releases]
Changelog:
==========
- Fixed: how taxes are computed for refunded orders.
- Cleaning Unit Tests
- Added: new events
- Added: disabling CSRF for webhook route
- Updated: module booting
- Added: generate barcode if not provided
- Added: checking customer coupons
- Fixed: fix #277
- Fixed: tax calculation on TaxService
- Fixed: fix #276
- Fixed: fix #274
- Updated: refactoring the code
- Added: Provider procurement Crud
- Added: support for quick products
- Updated: table schema for unit quantities
- Fixed: bug while making a payment
- FIxed: tax computation for quick product
- Fixed: tax computation while editing a product price
- Fixed: fix #278
- Fixed: notification not deleting
- Added: sound notifications
- Added: prevent undesired pos closure
- Added: new audio select input
- Added: customer account history crud component
- Fixed: allow custom payment for an installment

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS-4x/releases/tag/v4.5.0'>NexoPOS 4.5.0</a>.</em>

**Comment by Tex1000** [comment]
Pls fetch the customers rewards points in the pos too, just as we can see customers coupons. it is very important, right now we have to always go to the dashboard to see the customers points. It’s not convenient at all. All important data should be viewable on the pos when we open a customers profile
